src/components/CatalogOverlay/CatalogSubplotComponent/CatalogSubplotComponent.tsx

```diff
@@ -104,28 +134,43 @@ export class CatalogScatterComponent extends React.Component<WidgetProps> {
         CatalogStore.Instance.updateShowSelectedData(storeId, val);
     }
 
+    private handleLogScaleYChanged = (changeEvent: React.ChangeEvent<HTMLInputElement>) => {
+        const val = changeEvent.target.checked;
+        this.widgetStore.setLogScaleY(val);
+    }
+
     private onHover = (event: Plotly.PlotMouseEvent) => {
+        const widgetStore = this.widgetStore;        
         const points = event.points;
-        if (points.length) {
+        if (points.length && widgetStore) {
             const point = points[0];
-            this.widgetStore.setIndicator({x: point.x as number, y: point.y as number});  
+            widgetStore.setIndicator({x: point.x as number, y: point.y as number});
         }
     }
 
-    private genProfilerInfo = (): string[] => {
+    @computed get genProfilerInfo(): string[] {
         let profilerInfo: string[] = [];
         const widgetStore = this.widgetStore;
         const column = widgetStore.columnsName;
         const indicatorInfo = widgetStore.indicatorInfo;
         if (indicatorInfo) {
-            profilerInfo.push(column.x + ": " + indicatorInfo.x + ", " + column.y + ": " + indicatorInfo.y);   
+            if (widgetStore.plotType === CatalogPlotType.D2Scatter) {
+                profilerInfo.push(column.x + ": " + indicatorInfo.x + ", " + column.y + ": " + indicatorInfo.y);      
```

Comment:
probably cleaner to use ``    `${colum.x}: ${indicatorInfo.x}, ${column.y}: ${indicatorInfo.y}` `` and so on.

(p.s. working out how to get the character literal ` in code was fun)